src/pages/index.tsx

```diff
@@ -3,8 +3,18 @@ import { KeywordRadioBtnGroup } from "@/components/KeywordRadioBtnGroup/KeywordR
 import { ModeToggleBtn } from "@/components/ModeToggleBtn/ModeToggleBtn";
 import { Logo } from "@/components/Logo/Logo";
 import { Searchbar } from "@/components/Searchbar/Searchbar";
+import { useState } from "react";
 
 export default function Home() {
+  const [keyword, setKeyword] = useState("name");
+  const [text, setText] = useState("");
+  const [submitted, setSubmitted] = useState("");
+  const onSearchbarSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setSubmitted(text);
+    setText("");
+  };
```

Comment:
You shlould wrap all the functions in `React.useCallback`, otherwise the function will get redefined on every rerender
```suggestion
  const onSearchbarSubmit = React.useCallback((e: React.FormEvent<HTMLFormElement>) => {
    e.preventDefault();
    setSubmitted(text);
    setText("");
  }, []);
```

Reply:
done